coalib/misc/Exceptions.py

```diff
@@ -3,6 +3,8 @@
 from coalib.misc import Constants
 from coalib.output.printers.LogPrinter import LogPrinter
 
+from pkg_resources import ContextualVersionConflict, VersionConflict
```

Comment:
Scrutinizer fails saying the Contextual one doesnt exist.
I cant find in the docs either : https://pythonhosted.org/setuptools/pkg_resources.html#exceptions

Seems like the setuptools version is not high enough in scrutinizer ?
http://sickrage.ca/forums/forum/help-support/bug-issue-reports/33421-importerror-cannot-import-name-contextualversionconflict